cmd/init.go

```diff
@@ -38,180 +46,279 @@ func getInitCommand() *cobra.Command {
+			for _, template := range templates {
+				url := fmt.Sprintf("https://raw.githubusercontent.com/jarededwards/k3s-datacenter/refs/heads/main/templates/%s", template)
+				filename := filepath.Join(pwd, "templates/"+template)
 
-				_, err = exec.ExecuteCommand(log, "helm", installFlags...)
+				fmt.Println(filename)
+				err := download.FileFromURL(url, filename)
 				if err != nil {
-					return fmt.Errorf("error installing helm chart: %w", err)
+					return fmt.Errorf("error downloading file: %w", err)
+				} else {
+					log.Info("downloaded:", filename)
 				}
-				log.Info("Colony helm chart installed")
-			} else {
-				log.Info("Colony helm chart already installed")
+
+			}
+
+			manifests, err := readFilesInDir(filepath.Join(pwd, "templates"))
+			if err != nil {
+				return fmt.Errorf("error reading files directory: %w", err)
+			}
+
+			if err := k8sClient.ApplyManifests(ctx, manifests); err != nil {
+				return fmt.Errorf("error applying templates: %w", err)
+			}
+
+			clusterRoleName := "smee-role"
+
+			patch := []map[string]interface{}{
+				{
+					"op":   "add",
+					"path": "/rules/-",
+					"value": rbacv1.PolicyRule{
+						APIGroups: []string{"tinkerbell.org"},
+						Resources: []string{"hardware", "hardware/status"},
+						Verbs:     []string{"create", "update"},
+					},
+				},
+			}
+
+			// Convert patch to JSON
+			patchBytes, err := json.Marshal(patch)
+			if err != nil {
+				return fmt.Errorf("error marshalling clusterrole patch: %w", err)
+			}
+
+			err = k8sClient.PatchClusterRole(ctx, clusterRoleName, patchBytes)
+			if err != nil {
+				return fmt.Errorf("error patching ClusterRole: %w", err)
 			}
 
 			return nil
 		},
 	}
 
 	cmd.Flags().StringVar(&apiKey, "apiKey", "", "api key for interacting with colony cloud (required)")
-	cmd.Flags().StringVar(&apiURL, "apiURL", "", "api url for interacting with colony cloud (required)")
-	cmd.Flags().StringVar(&loadBalancer, "loadBalancer", "10.0.10.2", "load balancer ip address (required)")
+	cmd.Flags().StringVar(&apiURL, "apiURL", "https://colony-api-virtual.konstruct.io", "api url for interacting with colony cloud (required)")
+	cmd.Flags().StringVar(&loadBalancerInterface, "loadBalancerInterface", "eth0", "the local network interface for colony to use (required)")
+	cmd.Flags().StringVar(&loadBalancerIP, "loadBalancerIP", "192.168.0.192", "the local network interface for colony to use (required)")
 
 	return cmd
 }
 
 func init() {
 }
+
+func createDirIfNotExist(dir string) error {
+	if _, err := os.Stat(dir); errors.Is(err, fs.ErrNotExist) {
+		err = os.Mkdir(dir, 0o777)
+		if err != nil {
+			return fmt.Errorf("unable to create directory %q: %w", dir, err)
+		}
+	}
+	return nil
+}
+
+func readFilesInDir(dir string) ([]string, error) {
+
+	var templateFiles []string
+	// Open the directory
+	files, err := ioutil.ReadDir(dir)
```

Comment:
```suggestion
	files, err := os.ReadDir(dir)
```

`ioutil` is deprecated.